### PR TITLE
fix login and register as modal

### PR DESCRIPTION
**Pull request by tung3457**

### CODE REVIEW

**Comment by tung3457** [comment]
ready

File: resources/views/auth/login-modal.blade.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="modal-content modal-content-custom">
+            <div class="modal-header">
+                <h5 class="modal-title" id="loginModalTitle"><span>Login to continue</span></h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                {!! Form::open(['method' => 'POST', 'url' => route('login'), 'aria-label' => __('Login')]) !!}
+                {!! Form::email('email', old('email'), ['id' => 'email', 'placeholder' => 'Enter Email','class' => 'form-control' . ( $errors->has('email') ? ' is-invalid' : '' ), 'required' => 'required', 'autofocus' => 'autofocus']) !!}
```
Comment (BunnyPi04) [inline resources/views/auth/login-modal.blade.php]: convention

File: resources/views/auth/login-modal.blade.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                </div>
+                {!! Form::close() !!}
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary button-login-facebook" data-dismiss="modal">Login with facebook</button>
+                <button type="button" class="btn btn-primary button-login-google">Login with google</button>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (BunnyPi04) [inline resources/views/auth/login-modal.blade.php]: thiếu dòng trắng cuối file

File: resources/views/auth/login-modal.blade.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                {!! Form::submit(__('Login'), ['class' => 'btn btn-primary button-login']) !!}
+                <div class="text-center">
+                    <a class="link-custom" aria-busy="false" href="{{ route('password.request') }}">
+                        {!! trans('app.forgot-password') !!}
+                    </a>
+                </div>
+                {!! Form::close() !!}
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary button-login-facebook" data-dismiss="modal">Login with facebook</button>
```
Comment (BunnyPi04) [inline resources/views/auth/login-modal.blade.php]: i18n

File: resources/views/auth/register-modal.blade.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<!-- Modal -->
+<div class="modal fade" id="registerModal" tabindex="-1" role="dialog" aria-labelledby="registerModalTitle" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content modal-content-custom">
+            <div class="modal-header">
+                <h5 class="modal-title" id="loginModalTitle"><span>Register</span></h5>
```
Comment (BunnyPi04) [inline resources/views/auth/register-modal.blade.php]: i18n

File: resources/views/auth/register-modal.blade.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+                        <strong>{{ $errors->first('name') }}</strong>
+                    </span>
+                @endif
+                {!! Form::email('email', old('email'), ['id' => 'email', 'placeholder' => 'Enter Email' , 'class' => 'form-control' . ( $errors->has('email') ? ' is-invalid' : '' ), 'required' => 'required']) !!}
+                @if ($errors->has('email'))
+                    <span class="invalid-feedback" role="alert">
+                        <strong>{{ $errors->first('email') }}</strong>
+                    </span>
+                @endif
+                {!! Form::password('password', ['id' => 'password', 'placeholder' => 'Enter Password' , 'class' => 'form-control' . ( $errors->has('password') ? ' is-invalid' : '' ), 'required' => 'required']) !!}
```
Comment (BunnyPi04) [inline resources/views/auth/register-modal.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

File: resources/views/auth/login-modal.blade.php [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+                    </span>
+                @endif
+                {!! Form::password('password', ['id' => 'password', 'placeholder' => 'Enter Password', 'class' => 'form-control' . ( $errors->has('password') ? ' is-invalid' : '' ), 'required' => 'required']) !!}
+                @if ($errors->has('password'))
+                    <span class="invalid-feedback" role="alert">
+                        <strong>{{ $errors->first('password') }}</strong>
+                    </span>
+                @endif
+                {!! Form::checkbox('remember', '1', old('remember') ? true : false) !!}
+                {!! Form::label('remember', 'Remember Me') !!}
```
Comment (BunnyPi04) [inline resources/views/auth/login-modal.blade.php]: i18n

**Comment by BunnyPi04** [comment]
commented

**Comment by tung3457** [comment]
ready

**Pull request merged**